modules/hitlnext/src/backend/repository.ts

```diff
@@ -428,13 +441,14 @@ export default class Repository {
 
   updateHandoff = async (botId: string, id: string, attributes: Partial<IHandoff>) => {
     const now = new Date()
-    const payload = this.castDate(
+    let payload = this.castDate(
       {
         ...attributes,
         updatedAt: now
       },
       ['assignedAt', 'resolvedAt', 'updatedAt']
     )
+    payload = this.castJson(attributes, ['tags'])
```

Comment:
this won't work .... you frist add dates and serialize them. then serialize attributes and assign it back to payload ==> loosing the freshly created dates ?

Reply:
yeah, there seems to be an unused method there... payload is reassigned and castdate is not used

Reply:
Good catch, fixed.